src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java

```diff
@@ -4,16 +4,20 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import com.softserveinc.ita.pageobjects_task.kuguk.TestRunner;
 
 public class GoogleSearchResultsPage {
 
 	public List<String> getSearchResultsLinks() {
-
+		return 	Stream.of (TestRunner.driver.findElements(By.xpath("//h3[@class='LC20lb DKV0Md']")))  		
+				.map(foundLinksList -> foundLinksList.get(0).getText ()) 

```

Comment:
you're mapping each element onto first link text

Reply:
fixed